website/src/css/_navbar.scss

```
@@ -22,3 +22,29 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Restyle gamburger button */
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Restyle search button */
+
+@media (max-width: 769px) {
+  button.DocSearch-Button {
+    background-color: var(--detox-color-a3);
+    height: 40px;
+    border-radius: 10px;
+    padding: 0 10px;
+  }
```

SCSS syntax allows you to express the same intent via [nesting](https://www.w3schools.com/sass/sass_nesting.php) to avoid repetition:

```suggestion
    svg {
      opacity: 0.6;
    }
  }
```